evaluation/README.md

```diff
@@ -1 +1,87 @@
-# Evaluate your app
+# Evaluate Your AI App
+
+## Overall
+
+Evaluation is a critical process to measure the performance, quality and effectiveness of AI apps. This guide will help you to evaluate your AI apps and view the results in AI Toolkit.
```

Comment:
Evaluation is a critical process to measure the performance, quality, and effectiveness of AI apps. This guide will help you evaluate your AI apps and view the results in AI Toolkit.